### PR TITLE
IWD

### PR DESCRIPTION
**Pull request by harshpopat**
Corrected the name Surbhi Jain, added content about the organizers, added images, replaced the existing file.

### CODE REVIEW

File: _posts/2018-07-19-IWD.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+* Harvard CS 109 course in Data Science
+* Deep Learning Course on Udacity
+* Andrew's Machine Learning course on coursera
+
+### Women In Silicon Valley:
+#### Speaker: Darshil Bhatt
+There was a brief description on lack of women in tech as compared to men. This leaves them in minority which is not a good sign. Darshil proposed many ways to empower them and generate more participation to overcome gender inequality.
+
+### Dream it, Do it:
+#### Speaker: Saurabhi Jain
```
Comment (mistryhardik) [inline _posts/2018-07-19-IWD.md]: The name is **Surbhi Jain**.

File: _posts/2018-07-19-IWD.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# International Women's Day (IWD)
+
```
Comment (manan-jadhav) [inline _posts/2018-07-19-IWD.md]: add some more content here, explain when this was, where it was, who organised (WTM)

File: _posts/2018-07-19-IWD.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+### Keynote:
+#### Speaker: Mamta Gelanee
+The day dedicated to the wonderful women, International Women’s Day (IWD) began with a keynote sharing a brief description of the events stacked for the day including Women entrepreneurship and empowerment, data science stuff and many more informative sessions.
+
```
Comment (manan-jadhav) [inline _posts/2018-07-19-IWD.md]: add some images please

**Review by manan-jadhav — CHANGES_REQUESTED** [review]

**Comment by manan-jadhav** [comment]
Please add more descriptive PR title and description

**Comment by manan-jadhav** [comment]
You have created a new file instead of changing the existing one, please correct this as well.

File: _posts/2018-07-21-IWD.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
+# International Women's Day (IWD)
+
+A fabulous event organized by **WTM(Women Techmakers) held on 8th April 2018** was all about how far women have reached, where they are falling behind and how perceptions continue to shift. It was one of the memorable meets of [GDG Baroda](https://www.meetup.com/gdgbaroda/) to empower women.
+
+![IWD](https://scontent.fbdq1-1.fna.fbcdn.net/v/t1.0-9/30572137_951380235031037_4460753643510956032_n.jpg?_nc_cat=0&oh=0e09ffe70e44653aaddfc710f3f796ca&oe=5BCCDC63)
```
Comment (manan-jadhav) [inline _posts/2018-07-21-IWD.md:5]: Please download the images in the assets folder and use from there. These links can expire anytime.

File: _posts/2018-07-21-IWD.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
+# International Women's Day (IWD)
+
+A fabulous event organized by **WTM(Women Techmakers) held on 8th April 2018** was all about how far women have reached, where they are falling behind and how perceptions continue to shift. It was one of the memorable meets of [GDG Baroda](https://www.meetup.com/gdgbaroda/) to empower women.
```
Comment (manan-jadhav) [inline _posts/2018-07-21-IWD.md:3]: Please mention WTM Baroda

**Review by manan-jadhav — CHANGES_REQUESTED** [review]

**Comment by harshpopat** [comment]
Ok I'll do it, any further changes?

**Comment by harshpopat** [comment]
In the asset folder the images are of only tensorflow not IWD.

**Comment by harshpopat** [comment]
Ok bro I'll download in asset and then use it

**Pull request closed without merging**